frontend/src/index.css

```
@@ -5,6 +5,40 @@
   --font-sans:
     -apple-system, BlinkMacSystemFont, 'Apple SD Gothic Neo', 'Pretendard',
     'Malgun Gothic', system-ui, sans-serif;
+  --font-display:
+    'Iowan Old Style', 'Palatino Linotype', Palatino, 'Apple Myungjo',
```

Stylelint (17.14.0)

[error] Expected "Palatino" to be "palatino" (value-keyword-case)

(value-keyword-case)